app/controllers/art_items_controller.rb

```diff
@@ -1,4 +1,5 @@
 class ArtItemsController < ApplicationController
+  before_action :authenticate_user!
```

Comment:
Added by Rick to support the ID assignment in the Active Record annotation table

Reply:
Why did you push commits into my pull requests without even writing comments? They are doing nothing for the PR tasks but creating conflicts. Adding “before_action :authenticate_user!” should be done globally for the whole application, not only for this ticket. If you want to ‘support the ID assignment in the Active Record annotation table’ you should change the annotations table by adding “belongs_to :user” instead of ‘created_by :integer’ field that is linked to nothing. And what is the purpose of ’refactoring display of header links” in this PR?

Reply:
There is a ticket to setup authorization LRA-428. I did this here to allow for this code to be tested and evaluated.